features/tracker-allowlist.json

```
@@ -2564,6 +2564,17 @@
                     }
                 ]
             },
+            "synchrobox.adswizz.com": {
```

```suggestion
            "adswizz.com": {
```

and to go to appropriate location in file